MelodyGenerator.cpp

```diff
@@ -0,0 +1,25 @@
+
+#include "MelodyGenerator.h"
+
+MelodyGenerator::MelodyGenerator()
+{
+    std::cout << "Hello from MelodyGenerator struct, I am being constructed" << std::endl;
+    std::cout << "MG arpmodevalue = " << arpModeControlValue << std::endl;    arpeggiator.arpModeControlValue = 0;
+}
```

Comment:
Don't put more than one expression on a single line of code.
you have 2 expressions on line 7
- [ ] revise